dask/dataframe/utils.py

```diff
@@ -379,15 +379,15 @@ def _nonempty_index(idx):
         return typ([1, 2], name=idx.name)
     elif typ is pd.Index:
         return pd.Index(['a', 'b'], name=idx.name)
-    elif typ is pd.DatetimeIndex:
+    elif typ is pd.date_range:
```

Comment:
This won't work. `pd.date_range` is a function. Leaving DatetimeIndex should be fine.

Reply:
resolved.

Reply:
Is it okay now?